api/v1beta2/ocirepository_types.go

```diff
@@ -45,6 +45,9 @@ const (
 	// AzureOCIProvider provides support for OCI authentication using a Azure Service Principal,
 	// Managed Identity or Shared Key.
 	AzureOCIProvider string = "azure"
+
+	OCILayerExtract = "extract"
+	OCILayerCopy    = "copy"
```

Comment:
Would be good to have doc blocks on these two describing what behavior they enable.